larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc

```diff
@@ -298,6 +298,129 @@ StatusCode PfoHierarchyFeatureTool::ReadSettings(const TiXmlHandle /*xmlHandle*/
 //------------------------------------------------------------------------------------------------------------------------------------------
 //------------------------------------------------------------------------------------------------------------------------------------------
 
+ConeChargeFeatureTool::ConeChargeFeatureTool(): m_conMinHits(3), m_minCharge (0.1f), m_conFracRange(0.2f), m_MoliereRadius(10.1f), m_MoliereRadiusFrac(0.2f)
```

Comment:
Where does the default value for the Moliere radius come from? I thought this was about 9cm for liquid Argon

Reply:
I took it directly from the warwick pid: https://github.com/LArSoft/larana/blob/develop/larana/ParticleIdentification/MVAAlg.cxx#L546-L547
I cross-check on https://lar.bnl.gov/properties/#refs and I think it's about 10.0 for LAr @E=0.5kV